packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -358,6 +360,7 @@ class NavigatorState extends State<Navigator> {
       if (oldRoute != null)
         oldRoute.didChangeNext(route);
       config.observer?.didPush(route, oldRoute);
+      developer.postEvent('Flutter.Route', <String, dynamic>{ 'route': route.toString() });
```

Comment:
I that equivalent to a path I could use to restore the route? If no, I'll leave this out for now. I'd want to be able to show something like `/weather/daily`.